src/elements/content-preview/ContentPreview.js

```diff
@@ -158,6 +158,8 @@ class ContentPreview extends React.PureComponent<Props, State> {
 
     api: API;
 
+    contentSidebarRef: { current: null | ContentSidebar } = React.createRef();
```

Comment:
Should this also be just `contentSidebar` rather than `contentSidebarRef`?

Reply:
Good catch, I'll also update the sidebarPanelsRef